src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_create_with_http_proxy_config.yml

```diff
@@ -0,0 +1,606 @@
+interactions:
```

Comment:
Please rename this file with suffix `.yaml` instead of `.yml`. This should solve the problem of prompting `az login`. The testsdk (azure-cli-testsdk, further depends on azure-devtools) only recognizes recording files ending with `.yaml`. You may refer the code from [here](https://github.com/Azure/azure-python-devtools/blob/master/src/azure_devtools/scenario_tests/base.py#L123).

Reply:
oh thank you for that, hope thiss helps